packages/interpreter/src/interpreter.js

```diff
@@ -1,3 +1,5 @@
+import { setAttributeInner } from "./common.js";
```

Comment:
I believe this breaks liveview which injects the `WS_ADDR` prior to this file, causing "SyntaxError: import declarations may only appear at top level of a module"

Reply:
I'll take a look when I get home.

Reply:
#1073